virttest/utils_stress.py

```diff
@@ -86,8 +111,46 @@ def vm_stress_events(self, event, vm):
                                 'cur_config': self.current_vcpu,
                                 'cur_live': self.current_vcpu,
                                 'guest_live': self.current_vcpu}
-                    utils_hotplug.check_vcpu_value(vm, exp_vcpu, option="--live")
+                    utils_hotplug.check_vcpu_value(
+                        vm, exp_vcpu, option="--live")
             elif "reboot" in event:
                 vm.reboot()
+            elif "nethotplug" in event:
+                for iface_num in range(int(self.iface_num)):
+                    logging.debug("Try to attach interface %d" % iface_num)
+                    mac = utils_net.generate_mac_address_simple()
+                    options = ("%s %s --model %s --mac %s %s" %
+                               (self.iface_type, self.iface_source['network'],
+                                self.iface_model, mac, self.attach_option))
+                    logging.debug("VM name: %s , Options for Network attach: %s", vm.name, options)
+                    ret = virsh.attach_interface(vm.name, options,
+                                                 ignore_status=True)
+                    time.sleep(self.event_sleep_time)
+                    libvirt.check_exit_status(ret)
+                    if self.detach_option:
+                        options = ("--type %s --mac %s %s" %
+                                   (self.iface_type, mac, self.detach_option))
+                        logging.debug("VM name: %s , Options for Network detach: %s", vm.name, options)
+                        ret = virsh.detach_interface(vm.name, options,
+                                                     ignore_status=True)
+                        libvirt.check_exit_status(ret)
```

Comment:
@bssrikanth you can move all `result check` under `if not self.ignore_status:`

Reply:
sure

Reply:
I think you dont have to have a separate condition for it, virsh have the option to provide ignore_status, you can just pass the parameter when calling virsh.

Reply:
`libvirt.check_exit_status()` do the result check and failout, we do not need incertain conditions here as we care about multiple stress events...

Reply:
@sathnaga got you :+1: